src/nnbench/reporter/base.py

```diff
@@ -114,7 +115,7 @@ def display(
                 continue
             filteredctx = {
                 k: v
-                for k, v in flatten(ctx).items()
+                for k, v in Context._flatten_dict(ctx).items()
```

Comment:
I think we should `Context.make(bm["context"])` early, and then filter on it specifically. We can implement this together if you like.

Reply:
Happy to do that!